### ISSUE TITLE
Unable to import tags

### ISSUE THREAD

**Issue by kamicut** [Daniel-KM/Omeka-plugin-UpgradeToOmekaS#6]
Thank you for the amazing module, this has been a lifesaver! 

We're currently trying to upgrade our Omeka Classic 2.7 installation using this plugin. It works if we skip tagging, otherwise it fails with the following error when inserting the taggings into the database:

```
120  2021-09-30  01:18:50  Error  Core/Tags  Install module 

SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign 
key constraint fails (`omeka`.`tagging`, CONSTRAINT `FK_A4AED123BAD26311` FOREIGN KEY 
(`tag_id`) REFERENCES `tag` (`id`) ON DELETE SET NULL), query was: INSERT INTO `tagging` 
(`id`, `tag_id`, `resource_id`, `owner_id`, `status`, `created`, `modified`) 
VALUES (1, 1, 5549, NULL, 'approved', '2019-08-27 07:50:56', NULL),(2, 2, 5549, NULL, 'approved', '2019-08-27 07:53:40', NULL),
(518, 294, 1, NULL, 'approved', '2019-09-04 07:18:27', NULL),(520, 2, 3888, NULL, 'approved', '2019-09-04 08:01:31', NULL),
(521, 516, 3888, NULL, 'approved', '2019-09-04 08:01:31', NULL),(522, 517, 3888, NULL, 'approved', '2019-09-04 08:01:31', NULL),
(523, 129, 5542, NULL, 'approved', '2019-09-04 08:10:29', NULL),(524, 389, 5542, NULL, 'approved', '2019-09-04 08:10:29', NULL),
(525, 182, 5542, NULL, 'approved', '2019-09-04 08:10:29', NULL),(526, 326, 4, NULL, 'approved', '2019-09-11 12:40:50', NULL);
```

At first glance, this means that a tagging has been added where the tag it's referencing doesn't exist. Have you encountered anything similar?

## notes
1. We had to modify `libraries/UpgradeToOmekaS/Processor/AbstractCore.php` and set the maxVersion to `2.7` for the plugin to work
2. When we upgrade, we get an installation of Omeka-S v2.x, but Folksonomy requires v3.0, could this be the issue?

cc @majdal

**Comment by Daniel-KM** [comment]
Yes, it may be the issue, but not sure. I didn't update this module since a while, so i didn't checked all versions recently. 

Anyway, most of the code of this plugin will be removed soon, so the module will only install the omeka s core and modules, then the module Bulk Import will do the real database of omeka classic. In any cases, there may be issues with exact version of plugins.

**Comment by majdal** [comment]
Thanks for the info. But to make sure: the Bulk Import is [this one](https://github.com/Daniel-KM/Omeka-S-module-BulkImport), right?

It seems like it doesn't have an import tags functionality, or support for the Folksonomy module. Are we missing something? Is there a recommended way for importing tags, even if it's separate from the normal import processes?
